### ISSUE TITLE
VitTMatte pretrained models license

### ISSUE THREAD

**Issue by leo27heady** [hustvl/ViTMatte#9]
Hello,

I came across the ViTMatte repository, and I'm interested in utilizing the image matting model for commercial purposes. However, I have a few questions regarding its usage and fine-tuning for my specific task. I noticed that there are two different implementations available - one trained on the _Composition-1k_ dataset and the other on the _Distinctions-646_ dataset. I would appreciate some clarification regarding the following points:

1. **License**: The repository is under the MIT license, but I would like to confirm whether this license permits the commercial usage of the model and if there are any additional restrictions or considerations to be aware of.

2. **Commercial Usage**: Can I use the ViTMatte image matting model, as provided in the repository, for commercial purposes? Are there any specific conditions or requirements associated with commercial usage?

3. **Fine-tuning**: I intend to fine-tune the pretrained ViTMatte model on my specific task or dataset. Does the model's license allow me to modify and fine-tune it for my commercial use case? Are there any specific guidelines or provisions for fine-tuning that I should follow?

Thank you for providing this valuable image matting model and for clarifying the terms of its usage. I look forward to your response.

**Comment by JingfengYao** [comment]
Thanks for liking ViTMatte. Our repo is totally under the MIT license. Feel free to use our model weights as long as you follow MIT license. I Hope ViTMatte could help. 😄

**Comment by leo27heady** [comment]
Thanks for the response!

**Comment by Looki2000** [comment]
> Thanks for liking ViTMatte. Our repo is totally under the MIT license. Feel free to use our model weights as long as you follow MIT license. I Hope ViTMatte could help. 😄

The only problem is that [some photos in the COCO dataset are under Creative Commons NonCommercial license](https://github.com/cocodataset/cocoapi/issues/81), DINO pretrained model was trained on [ImageNet dataset which is "available for free to researchers for non-commercial use"](https://image-net.org/) and [MAE project "is under the CC-BY-NC 4.0 license"](https://github.com/facebookresearch/mae).

This makes these model weights completely incompatible with MIT license, and it cannot be used for commercial purposes. I spent a lot of time trying to find an image matting model that was not trained on non commercial datasets and I did not find one.
